particle-iot/particle-cli
# bugfix/ch67767/tinker-for-bsom

## Description

running the `particle flash --usb tinker` command when targeting a `bsom` fails with: `Error writing firmware: file does not exist and no known app found.` ([ch](https://app.clubhouse.io/particle/story/67767/particle-flash-usb-tinker-does-not-work-for-bsom-platform))


## How to Test

using a `bsom` device, run `particle flash --usb tinker` and verify that your device is flashed appropriately


## Completeness
